benchmark/benchmarkTests.go

```diff
@@ -171,3 +185,8 @@ func getSociProcess() (*SociProcess, error) {
 		sociConfig,
 		outputDir)
 }
+
+func getStargzProcess() (*StargzProcess, error) {
```

Comment:
Can you please address the concerns from linter?

Reply:
Done